tools/copy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Shell script equivalent of Metamath COPY tool
+# Mario Carneiro 7-Jan-2022
+
+usage() {
+  cat << HELP
+COPY - Similar to Unix "cat" but safe (same input & output name allowed)
+
+This command copies (concatenates) all input files in a space separated
+list to an output file.  The output file may have
+the same name as an input file.  Any previous version of the output
+file is renamed with a ~1 extension.
```

Comment:
The backup ~1 seems to have been forgotton.

Reply:
I'm just going to delete that line unless you have a good reason not to. Is that something to rely on in automated scripts?

Reply:
The automatic backup is a convenience feature for those using the tools manually and guards against unwanted data loss.  I do not care, this modification should be noted  somewhere though

Reply:
Is it easy to implement the safeRename and have an option --noSafeRename that can be used by scripts ?

Reply:
moderately easy, but I have lost motivation to keep adding epicycles on this PR.